main.py

```diff
@@ -12,8 +12,11 @@
 import subprocess
 import time
 import platform
+import requests
+import zipfile
 
 SETTINGS_FILE = './settings.json'
+settings = ""
```

Comment:
What is this?

Reply:
Now unused, forgot to remove, i was trying to access the settings outside of the main (no access to it if not passed through args), will be removed in next commit, same with typo's and odd naming's.